src/OmniSharp.Roslyn.CSharp/Services/Intellisense/IntellisenseService.cs

```diff
@@ -1,213 +0,0 @@
-using System;
-using System.Collections.Generic;
-using System.Composition;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.Completion;
-using Microsoft.CodeAnalysis.Recommendations;
-using Microsoft.CodeAnalysis.Text;
-using OmniSharp.Extensions;
-using OmniSharp.Mef;
-using OmniSharp.Models.AutoComplete;
-using OmniSharp.Options;
-using OmniSharp.Roslyn.CSharp.Services.Documentation;
-
-namespace OmniSharp.Roslyn.CSharp.Services.Intellisense
-{
-    [Obsolete("Please use CompletionService.")]
-    [OmniSharpHandler(OmniSharpEndpoints.AutoComplete, LanguageNames.CSharp)]
-    public class IntellisenseService : IRequestHandler<AutoCompleteRequest, IEnumerable<AutoCompleteResponse>>
```

Comment:
we cannot remove this, not all clients have switched to the completion endpoint (e.g. vim)

Reply:
This makes me very sad :(.

Reply:
I'm especially sad because I assumed I could and didn't expose the necessary things this service needed in external access.

Reply:
thanks a lot for bringing this back. We'll engage with folks to try to get them to move, it's just not there yet

Reply:
What is the minimum requirement to use the new endpoint instead of the old one in OmniSharp-vim? Can we make a couple of small changes and continue as we are, ignoring the extra code changes for now?

Someone has had a look at switching but we haven't had time to get into it properly. But if we can make a small change for the sake of keeping the server codebase lean, that is doable.

Reply:
> Can we make a couple of small changes and continue as we are, ignoring the extra code changes for now?

I expect so. The new endpoint is modeled on the `completion` LSP endpoint. The one concern would be override completion, as I assume that you won't really be able to consume the async completion prototype I've made.